.github/workflows/test-bot.yml

```diff
@@ -0,0 +1,23 @@
+name: brew test-bot
```

Comment:
Should we be skipping a name here as per the comments on https://github.com/alphagov/govuk-developer-docs/pull/2533 ?

Reply:
Sure, we can do. But I literally copied the bulk of this from the Homebrew auto-generated stuff so 🤷.